Project 3/style.css

```diff
@@ -0,0 +1,282 @@
+* {
+    font-family: "Inter", sans-serif;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 15px;
+}
+body {
+    height: 100vh;
```

Comment:
niepotrzebne ograniczenie :) przy rozdzielczości mobilnej widać, jak kontent wychodzi poza body :) w tym przypadku niczego to nie psuje, no ale to raczej fartem :)